docs/source/markdown/options/userns.pod.md

```diff
@@ -4,14 +4,14 @@
 ####> are applicable to all of those.
 #### **--userns**=*mode*
 
-Set the user namespace mode for all the containers in a pod. It defaults to the `PODMAN_USERNS` environment variable. An empty value ("") means user namespaces are disabled.
+Set the user namespace mode for all the containers in a pod. It defaults to the `PODMAN_USERNS` environment variable. An empty value ("") or **host** means user namespaces are disabled when podman is run rootful, and enabled when podman is rootless.
 
-Rootless user --userns=Key mappings:
+User --userns=Key mappings:
 
-Key       | Host User |  Container User
+Key       | Host User     |  Container User
 ----------|---------------|---------------------
-""        |$UID           |0 (Default User account mapped to root user in container.)
-keep-id   |$UID           |$UID (Map user account to same UID within container.)
+"" or host|$UID           | 0 (Default User account mapped to root user in container.)
+keep-id   |$UID           | $UID (Map user account to same UID within container.)
```

Comment:
```
# podman run --userns=keep-id alpine echo hi
Error: keep-id is only supported in rootless mode
# podman run --userns=nomap alpine echo hi
Error: nomap is only supported in rootless mode
```

Reply:
This is correct, but the next lines contain:
> The rootless option `--userns=keep-id` uses all the subuids and subgids of the user. 

> **keep-id**: creates a user namespace where the current rootless user's UID:GID are mapped to the same values in the container. This option is not allowed for containers created by the root user

> **nomap**: creates a user namespace where the current rootless user's UID:GID are not mapped into the container. This option is not allowed for containers created by the root user.

To be precise, this text:
> The rootless option `--userns=keep-id` uses all the subuids and subgids of the user.

is in fact present in userns.container.md and absent from userns.pod.md.

I think the above lines spell sufficiently clear, that `keep-id` and `nomap` are not allowed for rootful podman.